source/graphql/migrate-neurelo.txt

```diff
@@ -0,0 +1,32 @@
+.. _migrate-neurelo:
+
+==========================
+Migrate GraphQL to Neurelo
+==========================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to Neurelo.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Neurelo is a platform for developers designed to simplify the process of working
+with databases. It provides a powerful database abstraction with API-first
```

Comment:
This is a total nitpick which we may not be able to do anything about, but "powerful" here really grabbed me as an inappropriate marketing term to use in our docs. I'd love if we could lose powerful from here and the little blurb on the main "migrate" page. I understand if that's not possible, though.

```suggestion
with databases. It provides a database abstraction with API-first
```